package.json

```diff
@@ -1,28 +1,32 @@
 {
   "name": "console-plugin-template",
   "version": "0.0.1",
+  "description": "Template project for OpenShift Console dynamic plugins",
   "private": true,
-  "repository": "git@github.com:spadgett/console-plugin-template.git",
   "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/openshift/console-plugin-template.git"
+  },
   "scripts": {
     "clean": "rm -rf dist",
-    "build": "yarn clean && NODE_ENV=production yarn ts-node node_modules/.bin/webpack",
-    "build-dev": "yarn clean && yarn ts-node node_modules/.bin/webpack",
-    "start": "yarn ts-node node_modules/.bin/webpack serve",
+    "build": "yarn clean && NODE_ENV=production yarn webpack",
+    "build-dev": "yarn clean && yarn webpack",
```

Comment:
Not for this PR, but we do we `yarn clean` in these commands?

Nice that we can get rid of `ts-node` from the commands.

Reply:
`build` and `build-dev` scripts invoke a one-time webpack build.

Running `clean` ensures that the output `dist` directory doesn't contain any leftover artifacts.